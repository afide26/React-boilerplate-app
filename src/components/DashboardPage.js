import React from 'react';

const DashboardPage = ()=>{
  return(
    <div>
      <h1>Dashboard Page Contents</h1>
    </div>
  )
};

export default DashboardPage;