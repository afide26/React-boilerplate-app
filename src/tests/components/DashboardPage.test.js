import React from 'react';
import DashboardPage from './../../components/DashboardPage';
import { shallow } from 'enzyme';


test("should render the ExpenseListDashboard with children", ()=>{
  const wrapper = shallow(<DashboardPage/>);
  expect(wrapper).toMatchSnapshot();
})