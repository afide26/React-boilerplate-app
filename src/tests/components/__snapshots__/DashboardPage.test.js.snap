// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the ExpenseListDashboard with children 1`] = `
<div>
  <h1>
    Dashboard Page Contents
  </h1>
</div>
`;
